src/Umbraco.Headless.Client.Net/Delivery/Models/ContentFilter.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Umbraco.Headless.Client.Net.Delivery.Models
+{
+    public class ContentFilter
+    {
+        public ContentFilter(string contentTypeAlias, ContentFilterProperties[] properties)
```

Comment:
Since `contentTypeAlias` isn't required, maybe we should add an overload without that parameter?

Reply:
Yes, good point!